deleting-clusters-and-cleaning-up.md

```diff
@@ -0,0 +1,39 @@
+# Deleting clusters and cleaning up
+
+## Deleting clusters
+Clusters managed by Cluster API can be deleted in the same way as any other Kubernetes object.
+
+To delete an individual cluster by name run:
+
+```bash
+kubectl delete cluster docker-cluster-one
+```
+This delete operation ensures that the cluster and all of the objects that form part of the cluster e.g. Machines, Cluster Infrastructure, are correctly deleted. 
+
+To delete all clusters managed by Cluster API run:
+
+```bash
+kubectl delete clusters --all -A
+```
+
+## Cleaning up resources created by this tutorial
+The management cluster in this tutorial is not managed by Cluster API until [you make it self-hosted](./docker-cluster-self-hosted.md).
+
+To clean up the management cluster before this step run:
+
+```bash
+ kind delete cluster -n kubecon-na-22-capi-lab
+ ```
+
+All clusters created using the Docker infrastructure provider can be managed using kind. To clean up every Kubernetes clusters created in the tutorial - returning your system to the state [before initializing the management cluster](./your-first-cluster.md) run:
+
+```bash
+kind delete clusters --all
+```
+
+Additionally, the container registry created as part of initializing the management cluster should be stopped using:
+```bash
+docker stop kind-registry
+```
```

Comment:
The registry should not exist, right? (except if folks are using the script from the CAPI repo)